demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -3570,6 +3570,20 @@ export class DemoMeetingApp
         break;
     }
 
+    const chosenVideoSendCodec = (document.getElementById('videoCodecSelect') as HTMLSelectElement).value;
+    switch (chosenVideoSendCodec) {
+      case 'vp8':
+        this.videoCodecPreferences = [VideoCodecCapability.vp8()];
+        break;
+      case 'h264ConstrainedBaselineProfile':
+        this.videoCodecPreferences = [VideoCodecCapability.h264ConstrainedBaselineProfile(), VideoCodecCapability.vp8()];
```

Comment:
Does this also need `vp8()`? The case name says only `h264ConstrainedBaselineProfile`.

Reply:
Left a comment. `vp8` is just fallback value.